Gruntfile.js

```diff
@@ -3,11 +3,21 @@
 const fs = require('fs-extra');
 const path = require('path');
 const debug = require('debug');
+const os = require('os');
 
 const webpackConfig = require('./build/webpack.config');
 const webpackConfigTest = require('./test/test.webpack.config');
 const pkg = require('./package.json');
-const codeSignConfig = require('../keys/codesign');
+
+let codeSignConfig;
+if (os.platform() === 'darwin') {
+    try {
+        codeSignConfig = require('../keys/codesign');
+    } catch (err) {
```

Comment:
Remove this catch: if the option is not passed, it's better to fail hard, a person building the app is not expected to monitor build logs.

Reply:
As you've probably already noticed, I've kept the try/catch but only to be able to throw a more detailed Error with info on how to get around it if need be 🙂

<img src="https://user-images.githubusercontent.com/6006249/72679154-52ed7d00-3b11-11ea-8333-a45ef1d1bf33.png" width="697.5" />

Reply:
Yes noticed this, it makes sense indeed.